src/api/parser/src/index.js

```diff
@@ -0,0 +1,11 @@
+const { Satellite } = require('@senecacdot/satellite');
+const { router } = require('bull-board');
+const startParserQueue = require('./parser-queue');
+
+const service = new Satellite();
+
+startParserQueue();
```

Comment:
Let's comment this out so that we can land on prod/staging today, and not have it compete with the existing backend that is also running a Bull queue in Redis.